Beta_Integrations/ProofpointThreatResponse/ProofpointThreatResponse.yml

```diff
@@ -1,134 +1,740 @@
+      description: The threat name
+      type: String
+    - contextPath: ProofPointTRAP.Incident.events.description
+      description: The description of the event
+      type: String
+    - contextPath: ProofPointTRAP.Incident.events.malwareName
+      description: The malware name
+      type: String
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.alertSource
+      description: The alert source
+      type: String
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.startTime
+      description: The start time of the result
+      type: Date
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.endTime
+      description: The end time of the result
+      type: Date
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.status
+      description: The status of the result
+      type: String
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.recipientType
+      description: The recipient type
+      type: String
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.recipient
+      description: The recipient email address
+      type: String
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.messageId
+      description: The message ID
+      type: String
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.isRead
+      description: Weather the message has been read or not
+      type: Boolean
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.wasUndone
+      description: Weather the message was undone or not
+      type: String
+    - contextPath: ProofPointTRAP.Incident.quarantine_results.details
+      description: The details about the result
+      type: String
+    - contextPath: ProofPointTRAP.Incident.successful_quarantines
+      description: The number of seccessfull quarantines
+      type: Number
+    - contextPath: ProofPointTRAP.Incident.failed_quarantines
+      description: The number of failed quarantines
+      type: Number
+    - contextPath: ProofPointTRAP.Incident.pending_quarantines
+      description: The number of pending quarantines
+      type: Number
+    - contextPath: ProofPointTRAP.Incident.events.emails.body
+      description: The body of the email
+      type: String
+    - contextPath: ProofPointTRAP.Incident.events.emails.body_type
+      description: The format of the body
+      type: String
+    - contextPath: ProofPointTRAP.Incident.events.emails.headers
+      description: The headers of the emails
+      type: Unknown
+    - contextPath: ProofPointTRAP.Incident.events.emails.urls
+      description: Urls list from the email
+      type: Unknown
+    - contextPath: ProofPoint.Incident.event_ids
+      description: List of IDs attached to the incident
+      type: Unknown
+  - arguments:
+    - default: false
+      defaultValue: The id value of incident to add the comments.
+      description: The ID value of the incident to add the comment to
```

Comment:
I believe that the content of the `defaultValue` was meant to be in the `description`

Reply:
good catch